demos/multi_object_classification_pipeline/python/README.md

```diff
@@ -0,0 +1,87 @@
+# Multi Object Classification Demo {#ovms_demo_multi_object_classification_pipeline}
+This document demonstrates how to create complex pipelines using object detection and object recognition models from OpenVINO Model Zoo. As an example, we will use [ssdlite_mobilenet_v2](https://github.com/openvinotoolkit/open_model_zoo/blob/2022.1.0/models/public/ssdlite_mobilenet_v2/README.md) to detect objects on the image. Then, for each detected object we will crop it using [model_zoo_intel_object_detection](https://github.com/openvinotoolkit/model_server/tree/releases/2022/1/src/custom_nodes/model_zoo_intel_object_detection) example custom node. Finally, each object image will be forwarded to [efficientnet-b0](https://github.com/openvinotoolkit/open_model_zoo/blob/2022.1.0/models/public/efficientnet-b0/README.md) model.
+
+TODO: Add visualisation
+
+Using such pipeline, a single request to OVMS can perform a complex set of operations to determine all objects and their classes.
+
+## Pipeline Configuration Graph
+
+Below is depicted graph implementing multi object classification pipeline execution. 
+
+TODO: Add visualisation
+
+It includes the following Nodes:
+- Model `ssdlite_mobilenet_v2` - deep learning model which takes user image as input. Its outputs contain information about object coordinates and confidence levels.
+- Custom node `model_zoo_intel_object_detection` - it includes C++ implementation of common object detection models results processing. By analysing the output it produces cropped object images based on the configurable score level threshold. Custom node also resizes them to the target resolution and combines into a single output of a dynamic batch size. The output batch size is determined by the number of detected
+boxes according to the configured criteria. All operations on the images employ OpenCV libraries which are preinstalled in the OVMS. Learn more about the [model_zoo_intel_object_detection custom node](https://github.com/openvinotoolkit/model_server/tree/releases/2022/1/src/custom_nodes/model_zoo_intel_object_detection).
+- demultiplexer - outputs from the custom node model_zoo_intel_object_detection have variable batch size. In order to match it with the sequential `efficientnet-b0` models, data is split into individuial images with each batch size equal to 1.
+Such smaller requests can be submitted for inference in parallel to the next Model Nodes. Learn more about the [demultiplexing](../../../docs/demultiplexing.md).
```

Comment:
Maybe add note here that you can use dynamic shapes if following model does support it?

Reply:
They would need to make changes in custom node src